i18n4go/i18n/init.go

```diff
@@ -198,11 +195,15 @@ func translate(loc *go_i18n.Localizer) TranslateFunc {
 			}
 		}
 
-		return loc.MustLocalize(&go_i18n.LocalizeConfig{
+		msg, err := loc.Localize(&go_i18n.LocalizeConfig{
 			MessageID:    messageId,
 			TemplateData: data,
 			PluralCount:  count,
 		})
+		if err != nil {
+			return ""
+		}
+		return msg
```

Comment:
**Callout**: Instead of panic if there is a missing translation we will just not translate it in the source code.